vpn/options.go

```diff
@@ -407,11 +514,14 @@ func hasElement(el string, arr []string) bool {
 	return false
 }
 
+// existsFile returns true if the file to which the path refers to exists.
 func existsFile(path string) bool {
 	_, err := os.Stat(path)
 	return !errors.Is(err, os.ErrNotExist)
```

Comment:
```suggestion
	statbuf, err := os.Stat(path)
	return !errors.Is(err, os.ErrNotExist) && statbuf.Mode().IsRegular()
```

(You don't want this function to return true for an existing directory)

Reply:
fixed